src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        String split = ",";
```

Comment:
Since the comma ',' is used as a delimiter in both reading and writing, it should be declared as a constant with a meaningful name, as per the checklist item 'Use constants'.